verification/sign.go

```diff
@@ -0,0 +1,170 @@
+// Licensed under the Apache-2.0 license
+
+package verification
+
+import (
+	"bytes"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/x509"
+	"errors"
+	"math/big"
+	"testing"
+)
+
+// Obtain and validate signature of asymmetric signing.
+// Check whether the digital signature returned by Sign command can be verified
+// using public key in signing key certificate returned by CertifyKey command.
+// Inspite of the DPE profile supporting symmetric key, for symmetric signing it must be enabled
+// explicitly in Sign command flags. Else asymmetric signing is used as default.
+func TestAsymmetricSigning(d TestDPEInstance, c DPEClient, t *testing.T) {
```

Comment:
Checks to ensure that the data signed by DPE is verifiable through ecdsa.verify()